openlibrary/macros/BookPreview.html

```diff
@@ -1,11 +1,13 @@
-$def with (ocaid, analytics_attr, show_only=False)
+$def with (ocaid, analytics_attr=None, show_only=False)
 $# :param str ocaid:
 
+$ analytics = analytics_attr('Preview') if analytics_attr else "CTAClick|Preview"
```

Comment:
```suggestion
$ analytics = analytics_attr('Preview') if analytics_attr else 'data-ol-link-track="CTAClick|Preview"'
```
Should fix incorrect analytics data attribute:
![image](https://github.com/user-attachments/assets/961406da-a31a-4b4d-b655-a1eff31756d6)